<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.17">
  <compounddef id="classConfigurationException" kind="class" language="C++" prot="public">
    <compoundname>ConfigurationException</compoundname>
    <basecompoundref refid="cpp/error/exception" prot="public" virt="non-virtual">std::exception</basecompoundref>
      <sectiondef kind="private-attrib">
      <memberdef kind="variable" id="classConfigurationException_1ab0852d61165c8f4499c4ad2c4575df70" prot="private" static="no" mutable="no">
        <type><ref refid="cpp/string/basic_string" kindref="compound" external="/home/docker/.local/lib/python3.8/site-packages/catkin_tools_document/external/cppreference-doxygen-web.tag.xml">std::string</ref></type>
        <definition>std::string ConfigurationException::message_</definition>
        <argsstring></argsstring>
        <name>message_</name>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="src/localization_plugin.cpp" line="39" column="17" bodyfile="src/localization_plugin.cpp" bodystart="39" bodyend="-1"/>
      </memberdef>
      </sectiondef>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="classConfigurationException_1ad08edf483f9f86287c7687b74451f386" prot="public" static="no" const="no" explicit="no" inline="yes" virt="non-virtual">
        <type></type>
        <definition>ConfigurationException::ConfigurationException</definition>
        <argsstring>(const std::string &amp;msg)</argsstring>
        <name>ConfigurationException</name>
        <param>
          <type>const <ref refid="cpp/string/basic_string" kindref="compound" external="/home/docker/.local/lib/python3.8/site-packages/catkin_tools_document/external/cppreference-doxygen-web.tag.xml">std::string</ref> &amp;</type>
          <declname>msg</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="src/localization_plugin.cpp" line="32" column="5" bodyfile="src/localization_plugin.cpp" bodystart="32" bodyend="32"/>
      </memberdef>
      <memberdef kind="function" id="classConfigurationException_1ab3535f7c8b14e9c689a05436a3a6ba9c" prot="public" static="no" const="yes" explicit="no" inline="yes" virt="virtual">
        <type>const char *</type>
        <definition>virtual const char* ConfigurationException::what</definition>
        <argsstring>() const</argsstring>
        <name>what</name>
        <exceptions> throw ()</exceptions>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="src/localization_plugin.cpp" line="34" column="24" bodyfile="src/localization_plugin.cpp" bodystart="34" bodyend="37"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
    </detaileddescription>
    <inheritancegraph>
      <node id="2">
        <label>std::exception</label>
        <link refid="cpp/error/exception" external="/home/docker/.local/lib/python3.8/site-packages/catkin_tools_document/external/cppreference-doxygen-web.tag.xml"/>
      </node>
      <node id="1">
        <label>ConfigurationException</label>
        <link refid="classConfigurationException"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="2">
        <label>std::exception</label>
        <link refid="cpp/error/exception" external="/home/docker/.local/lib/python3.8/site-packages/catkin_tools_document/external/cppreference-doxygen-web.tag.xml"/>
      </node>
      <node id="1">
        <label>ConfigurationException</label>
        <link refid="classConfigurationException"/>
        <childnode refid="2" relation="public-inheritance">
        </childnode>
        <childnode refid="3" relation="usage">
          <edgelabel>message_</edgelabel>
        </childnode>
      </node>
      <node id="3">
        <label>std::string</label>
        <link refid="cpp/string/basic_string" external="/home/docker/.local/lib/python3.8/site-packages/catkin_tools_document/external/cppreference-doxygen-web.tag.xml"/>
      </node>
    </collaborationgraph>
    <location file="src/localization_plugin.cpp" line="30" column="1" bodyfile="src/localization_plugin.cpp" bodystart="29" bodyend="40"/>
    <listofallmembers>
      <member refid="classConfigurationException_1ad08edf483f9f86287c7687b74451f386" prot="public" virt="non-virtual"><scope>ConfigurationException</scope><name>ConfigurationException</name></member>
      <member refid="cpp/error/exception/exception_1" prot="public" virt="non-virtual"><scope>ConfigurationException</scope><name>exception</name></member>
      <member refid="classConfigurationException_1ab0852d61165c8f4499c4ad2c4575df70" prot="private" virt="non-virtual"><scope>ConfigurationException</scope><name>message_</name></member>
      <member refid="cpp/error/exception/operator=_1" prot="public" virt="non-virtual"><scope>ConfigurationException</scope><name>operator=</name></member>
      <member refid="classConfigurationException_1ab3535f7c8b14e9c689a05436a3a6ba9c" prot="public" virt="virtual"><scope>ConfigurationException</scope><name>what</name></member>
      <member refid="cpp/error/exception/what_1" prot="public" virt="non-virtual" ambiguityscope="std::exception::"><scope>ConfigurationException</scope><name>what</name></member>
      <member refid="cpp/error/exception/~exception_1" prot="public" virt="non-virtual"><scope>ConfigurationException</scope><name>~exception</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
